packages/crypto/src/utils.ts

```diff
@@ -1,3 +1,5 @@
+import type { Jwk } from './jose/jwk.js';
```

Comment:
is there some initiative to eject from jose/jwk in web5-js, or is ejecting from dependency effort just for web5-kt?

Reply:
Just web5-let in the context of JOSE (JWK, JWS, JWE, JWT) and ejecting from external deps.

There is an initiative that underway to consolidate the aforementioned JOSE functionality into the `@web5/crypto` package.  That entails migrating JWT from `@web5/credentials` and completing implementation of JWS and JWE.  JWK already lives in `crypto`.  I’ve got a draft of Compact and Flattened JWE just about done so will open a PR for those soon.